### PR TITLE
feat: Adding chat store init methods.

### PR DESCRIPTION
**Pull request by dishaprakash**
feat: Adding chat store init methods.

### CODE REVIEW

**Review by vishwarajanand — APPROVED** [review]
LGTM

File: src/llama_index_alloydb_pg/engine.py [hunk 1]
```diff
@@ -767,6 +767,98 @@ def init_index_store_table(
...
+            async with self._pool.connect() as conn:
+                await conn.execute(
+                    text(f'DROP TABLE IF EXISTS "{schema_name}"."{table_name}"')
+                )
+                await conn.commit()
+
+        create_table_query = f"""CREATE TABLE "{schema_name}"."{table_name}"(
+            id SERIAL PRIMARY KEY,
+            key VARCHAR NOT NULL,
+            messages JSON[] NOT NULL,
```
Comment (averikitsch) [inline src/llama_index_alloydb_pg/engine.py:799]: Message is just JSON not an array.

File: src/llama_index_alloydb_pg/engine.py [hunk 2]
```diff
@@ -767,6 +767,98 @@ def init_index_store_table(
...
+            CONSTRAINT "{table_name}_unique_key" UNIQUE (key)
```
Comment (averikitsch) [inline src/llama_index_alloydb_pg/engine.py:800]: The key will not be unique. Each key will have mulitple rows with 1 message per row with an unique id for ordering

File: src/llama_index_alloydb_pg/engine.py (hunk 1 above)
Comment (averikitsch) [inline src/llama_index_alloydb_pg/engine.py:799]: This should be message not messages. Message is just JSON not an array.

**Review by averikitsch — CHANGES_REQUESTED** [review]

**Review by averikitsch — APPROVED** [review]

**Pull request merged**